### ISSUE TITLE
Can High side Outputs drive VVT solenoids

### ISSUE THREAD

**Issue by ElDominio** [rusefi/alphax-4chan#173]
Ask question states, can high side output drive a VVT solenoid that has one side wired to ground?

**Comment by rusefillc** [comment]
@ElDominio please provide context what pins are you asking about? ignition control I am sure is too weak

**Comment by ElDominio** [comment]
Internal High1 and High2, the ones on the molex microfit

**Comment by Qwerty-OFF** [comment]
Yes, we need an experiment.  Perhaps long-term operation with a current of 3 Amps will cause significant heating.  These outputs are unprotected.  Flyback diodes are present, you can control the inductive load using PWM.

**Comment by ElDominio** [comment]
I don't think VVT solenoids ask for more than half an amp, so we should be fine?

**Comment by rusefillc** [comment]
> we need an experiment

@ElDominio would you be able to make an experiment happen?

**Comment by ElDominio** [comment]
What is the experiment

**Comment by rusefillc** [comment]
@ElDominio please wire up a VVT solenoid on bench or real car and see what happens

**Comment by ElDominio** [comment]
This will be tested eventually

**Comment by ElDominio** [comment]
It may not be because #176 happened
